src/main/scala/com/raphtory/output/FileOutputFormat.scala

```diff
@@ -8,17 +8,41 @@ import org.slf4j.LoggerFactory
 import java.io.File
 
 /**
-  * # FileOutputFormat
-  * `FileOutputFormat` writes output for Raphtory Job and Partition for a pre-defined window and timestamp to File
+  * {s}`FileOutputFormat(filePath: String)`
+  *
+  *   : writes output for Raphtory Job and Partition for a pre-defined window and timestamp to File
+  *        {s}`filePath: String`
+  *          : Filepath for writing Raphtory output.
+  *   Reference: [](com.raphtory.core.algorithm.OutputFormat)
```

Comment:
This is not indented correctly and thus does not produce the expected output, please check!